packages/sign-client/package.json

```diff
@@ -27,7 +27,7 @@
     "test:concurrency": "vitest run --dir test/concurrency",
     "test:xregion": "vitest run --dir test/xregion",
     "test": "npm run test:pre; npm run test:run",
-    "test:ignoreUnhandled": "npm run test:pre; npm run test:run -- --dangerouslyIgnoreUnhandledErrors",
+    "test:ignoreUnhandled": "npm run test:pre; npm run test:run -- --dangerouslyIgnoreUnhandledErrors --no-threads",
```

Comment:
how much slower is this making tests?

Reply:
Comparing previous CI runs

- [3m 21s](https://github.com/WalletConnect/rs-relay/actions/runs/3450693108/jobs/5778391203) - old
- [3m 30s](https://github.com/WalletConnect/rs-relay/actions/runs/3574722287/jobs/6012510889) - new
I would say marginally